### PR TITLE
Lecture on AIS

### PR DESCRIPTION
**Pull request by kamakshi94**
### What

It's a pull request for the lecture on Artificial Immune System

### Why
For tomorrow lecture

###Where
Place the content on website under student lecture

### CODE REVIEW

File: site/student-lectures/AIS/ais.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
+.. _lecture:
+
+Artificial Immune System
+===============================
+
+**What is Artificial Immune Systems?**
+---------------------------------------
```
Comment (jameshughes89) [inline site/student-lectures/AIS/ais.rst:7]: underlines too long

File: site/student-lectures/AIS/ais.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+   problem-solving.
+
+   AIS emerged in the mid-1980s with articles authored by Farmer,
+   Packard and Perelson (1986) and Bersini and Varela (1990) on immune
+   networks. However, it was only in the mid-1990s that AIS became a
+   field in its own right.
+
+   Forrest et al. (on negative selection) and Kephart et al.[2]
+   published their first papers on AIS in 1994, and Dasgupta conducted
+   extensive studies on Negative Selection Algorithms.
```
Comment (jameshughes89) [inline site/student-lectures/AIS/ais.rst:22]: You have taken this verbatim from wikipedia.   

https://en.wikipedia.org/wiki/Artificial_immune_system

File: site/student-lectures/AIS/ais.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+   algorithms have been used in clustering, data visualization, control,
+   and optimization domains, and share properties with artificial neural
+   networks.
+
+**4.Dendritic cell algorithm:** The DCA is abstracted and
+   implemented through a process of examining and modeling various
+   aspects of DC function, from the molecular networks present within
+   the cell to the behaviour exhibited by a population of cells as a
+   whole. Within the DCA information is granulated at different layers,
+   achieved through multi-scale processing.
```
Comment (jameshughes89) [inline site/student-lectures/AIS/ais.rst:93]: This too is taken directly from wikipedia 
https://en.wikipedia.org/wiki/Artificial_immune_system

File: site/student-lectures/AIS/ais.rst [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+
+**Clonal Selection Algorithm**
+----------------------------------
```
Comment (jameshughes89) [inline site/student-lectures/AIS/ais.rst:97]: too long

File: site/student-lectures/AIS/ais.rst [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+.. image:: \image006.png
+   :alt: Clonal Algorithm
+   :align: center
+
+* **Initial population:** The first step is the initial population valuation. Each member of this set is called a chromosome.In this step a set of valid solutions for the problem is produced randomly, additionally, there is a memory that is called M and it exists beside this set. In each execution cycle, algorithm saves the best founded solution on its memory.
```
Comment (jameshughes89) [inline site/student-lectures/AIS/ais.rst:102]: https://www.elsevier.es/en-revista-journal-applied-research-technology-jart-81-articulo-proposing-features-preprocessing-method-based-S1665642315000486

**Review by jameshughes89 — CHANGES_REQUESTED** [review]
I stopped reviewing your PR
[The first portion is copied directly from wikipedia](https://en.wikipedia.org/wiki/Artificial_immune_system)
[The next portion is taken from this article ](https://www.elsevier.es/en-revista-journal-applied-research-technology-jart-81-articulo-proposing-features-preprocessing-method-based-S1665642315000486)

Further, did you check the build of the website to ensure things formatted correctly?

File: site/student-lectures/AIS/ais.rst (hunk 2 above)
Comment (kamakshi94) [reply site/student-lectures/AIS/ais.rst:22]: i have added it as a reference

File: site/student-lectures/AIS/ais.rst (hunk 3 above)
Comment (kamakshi94) [reply site/student-lectures/AIS/ais.rst]: I have changed it in my words

File: site/student-lectures/AIS/ais.rst (hunk 5 above)
Comment (kamakshi94) [reply site/student-lectures/AIS/ais.rst:86]: added this journal in references

**Pull request closed without merging**